README.md

```diff
@@ -141,22 +120,45 @@ load_module modules/ngx_curity_http_phantom_token_module.so;
 
 The file can be an absolute or relative path. If it is not absolute, it should be relative to the NGINX root directory.
 
+### NGINX Parameters for the Introspection Endpoint
+
+You must also configure standard NGINX parameters for the introspection request:
+
```

Comment:
This is the main impact for the customer:

- Keeps the subrequest from receiving the upstream API's headers
- For that to work, we also need to configure the authorization header